homeassistant/helpers/entity.py

```diff
@@ -425,6 +428,20 @@ def _name_translation_key(self) -> str | None:
             f".{self.translation_key}.name"
         )
 
+    def _name_placeholders(self, name: str) -> str:
+        """Process possible placeholders in entity name."""
+        tuples = list(string.Formatter().parse(name))
+        if tuples[0][1] is None:
+            return name
```

Comment:
The checks here seem unnecessary, we could make the default return value of `translation_placeholders` an empty dictionary and unconditionally call `name.format` below.

Reply:
The checks are removed